src/platform/platform_winuser.c

```diff
@@ -266,12 +266,18 @@ CxPlatProcessorInfoInit(
     CXPLAT_FREE(Buffer, QUIC_POOL_PLATFORM_TMP_ALLOC);
```

Comment:
freeing Buffer will have the same issue.

Reply:
Fixed